common.tsx

```diff
@@ -88,6 +95,48 @@ export const md = markdownit({
   html: true,
 }).use(markdownitKatex);
 
+const postHref = (id: string) => `/blog/${id}`;
+const escapeXML = (str: string) =>
+  str.replace(
+    /[&<>'"]/g,
+    (tag) =>
+      ({
+        "&": "&amp;",
+        "<": "&lt;",
+        ">": "&gt;",
+        "'": "&#39;",
+        '"': "&quot;",
+      })[tag] ?? "",
+  );
+
+const rss = (posts: PublishedPost[], baseUrl: string) => {
+  const items = posts
+    .map(
+      ([id, p]) => `<item>
+<title>${escapeXML(p.title)}</title>
+<description>TODO</description>
+<link>${new URL(postHref(id), baseUrl).href}</link>
+<guid>${new URL(postHref(id), baseUrl).href}</guid>
+<pubDate>${new Date(p.date).toUTCString()}</pubDate>
+</item>`,
+    )
+    .join("\n");
+
+  const feedUrl = new URL("rss.xml", baseUrl).href;
+  const feed = `<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+<channel>
+<atom:link href="${feedUrl}" rel="self" type="application/rss+xml" />
```

Comment:
I don't see this element mentioned in the [spec](https://www.rssboard.org/rss-specification); but I'm new to RSS, so I'm guessing I'm missing something. I see that the rssboard.org site also links to [RFC 4287](https://datatracker.ietf.org/doc/html/rfc4287) for "The Atom Syndication Format" which does mention this element type; what's the relationship between RSS and Atom?